src/interceptors/on-request-refresh-token.ts

```diff
@@ -0,0 +1,55 @@
+import { AxiosError, InternalAxiosRequestConfig } from 'axios';
+import { decode } from 'base-64';
+import { SessionExpiryRefreshInterceptorArgs } from '../types';
+import { checkIsTokenExpired } from '../utils';
+
+global.atob = decode;
+
+let refreshTokenRequest: Promise<string> | null;
+
+export const onRequestSessionExpiryRefreshInterceptor =
```

Comment:
Could you please rename this interceptor and types to match file name?

`onRequestRefreshToken`